package/samplers/catcma/README.md

```diff
@@ -78,7 +85,7 @@ if __name__ == "__main__":
 
 Ryoki Hamano, Shota Saito, Masahiro Nomura, Kento Uchida, Shinichi Shirakawa , CatCMA : Stochastic Optimization for Mixed-Category Problems, GECCO'24
 
-See the [paper](https://arxiv.org/abs/2405.09962) for more details.
+See the [arXiv paper](https://arxiv.org/abs/2405.09962) or [ACM paper](https://dl.acm.org/doi/abs/10.1145/3638529.3654198) for more details.
```

Comment:
How about using the DOI  as shown in the ACM page.
<img width="973" alt="image" src="https://github.com/user-attachments/assets/994a9929-cc6b-45e2-9da9-763c13a71fe6">
 
```suggestion
See the [arXiv paper](https://arxiv.org/abs/2405.09962) or [ACM paper](https://doi.org/10.1145/3638529.3654198) for more details.
```